app/components/Modal/DepositModal.jsx

```diff
@@ -81,10 +82,34 @@ class DepositModalContent extends DecimalChecker {
             selectedGateway: null,
             fetchingAddress: false,
             backingAsset: null,
-            gatewayStatus: availableGateways
+            gatewayStatus: availableGateways,
+            depositConfirmation: null
         };
     }
 
+    _getDepositConfirmation(backingAsset) {
```

Comment:
This section should use non-gateway specific translation strings.

Reply:
Fixed